pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/ufoer/Etteroppgjoer.kt

```diff
@@ -0,0 +1,231 @@
+                English to "In some cases we carry out several settlements for the same year. All settlements apply. We compare them to see whether you must repay money, or whether you are owed any money."
+            )
+        }
+    }
+}
+
+object MeldeFraOmEndringerEtteroppgjoer : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        title1 {
+            text(
+                Bokmal to "Du må melde fra om endringer",
+                Nynorsk to "Husk å melde frå om endringar",
+                English to "You must report any changes"
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "For at du skal få utbetalt riktig uføretrygd fremover, er det viktig at du oppdaterer inntekten din. Dette gjør du på nav.no/inntektsplanleggeren. I vedlegget «Dine rettigheter og plikter» ser du hvilke endringer du må si fra om.",
+                Nynorsk to "For at du skal få utbetalt rett uføretrygd framover, er det viktig at du oppdaterer inntekta di. Dette gjer du på nav.no/inntektsplanleggeren. I vedlegget «Rettane og pliktene dine» ser du kva endringar du må seie frå om.",
+                English to "To be paid the correct amount of disability benefit in the future, it is important that you update your income. You can do this at: nav.no/inntektsplanleggeren. In the appendix «Your rights and obligations» you can see which changes you need to report."
+            )
+        }
+    }
+}
+
+object FristerOpplysningerKlage : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        title1 {
+            text(
+                Bokmal to "Frister for å sende inn nye opplysninger og klage",
+                Nynorsk to "Fristar for å sende inn nye opplysningar og klage",
+                English to "Deadlines for sending in new information and appeals"
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "Hvis du mener at beregningene i vedlegg «Beregning av etteroppgjøret» er feil, må du melde fra til oss innen 3 uker fra du fikk dette brevet. Du vil da få en ny vurdering og et nytt vedtak.",
+                Nynorsk to "Dersom du meiner at utrekningane i vedlegg «Beregning av etteroppgjøret» er feil, må du melde frå til oss innan 3 veker frå du fekk dette brevet. Du vil då få ei ny vurdering og eit nytt vedtak.",
+                English to "If you believe that the calculations in appendix «Calculation of settlement» are incorrect, you must notify us within 3 weeks of the date you received this letter. In such case you will receive a new assessment and a new decision."
+            )
+        }
+        paragraph {
+            text(
```

Comment:
Bruk `Constants.KLAGE_URL` her. Gjelder alle tre språk.

Reply:
Denne må fremdeles fikses

Reply:
Lagt in Constants.KLAGE_URL